src/server/on-off-farm/index.js

```diff
@@ -0,0 +1,36 @@
+import {
+  onOffFarmGetController,
+  onOffFarmPostController
+} from './controller.js'
+
+/**
+ * Sets up the routes used in the home page.
+ * These routes are registered in src/server/router.js.
+ */
+
+/**
+ * @satisfies {ServerRegisterPluginObject<void>}
+ */
+export const onOffFarm = {
```

Comment:
There may be a reason for this, but this could be easily cut down to just a list of routes, and then in server.js we can call `service.routes([...onOffFarmRoutes])`.

This one's a comment rather than a request for change.